### PR TITLE
Refactor prometheus alerts. Lint and check robocop rules

### PR DESCRIPTION
**Pull request by jgarciao**

### CODE REVIEW

**Comment by bdattoma** [comment]
lgtm

**Review by bdattoma — DISMISSED** [review]

**Comment by jgarciao** [comment]
Latest version tested successfully here: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/442

**Comment by jgarciao** [comment]
Tested successfully here: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/442

File: tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot [hunk 1]
```diff
@@ -1,74 +1,86 @@
...
     Maybe Close Popup
-    Navigate Home (Root folder) In JupyterLab Sidebar File Browser
-    Delete Folder In User Notebook  ${OCP_ADMIN_USER.USERNAME}  ${TEST_USER.USERNAME}  ods-ci-notebooks-main
-    Sleep  10  reason=Waiting for possible JupyterLab pop-up
+    Navigate Home (Root Folder) In JupyterLab Sidebar File Browser
+    Delete Folder In User Notebook    ${OCP_ADMIN_USER.USERNAME}    ${TEST_USER.USERNAME}    ods-ci-notebooks-main
+    Sleep    10    reason=Waiting for possible JupyterLab pop-up
     Maybe Close Popup
-    Sleep  10  reason=Waiting for possible JupyterLab pop-up
+    Sleep    10    reason=Waiting for possible JupyterLab pop-up
```
Comment (tarukumar) [inline tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot:69]: I think it will be good idea to put "s" for seconds  after the digit

**Review by tarukumar — APPROVED** [review]
Overall looks good to me. I have added one comment

**Review by bdattoma — APPROVED** [review]

File: tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot (hunk 1 above)
Comment (jgarciao) [reply tests/Tests/200__monitor_and_manage/200__metrics/203__alerts.robot]: Fixed

**Review by tarukumar — APPROVED** [review]
LGTM

**Review by bdattoma — APPROVED** [review]

**Pull request merged**